### ISSUE TITLE
Order of axes for `CuImage.origin`?

### ISSUE THREAD

**Issue by jakirkham** [rapidsai/cucim#709]
What is the order of the axes for [`CuImage.origin`]( https://docs.rapids.ai/api/cucim/stable/api/#cucim.clara.CuImage.origin )? Is it C-order or F-order? Also does it have any relationship to [`CuImage.dims`]( https://docs.rapids.ai/api/cucim/stable/api/#cucim.clara.CuImage.dims )

cc @thewtex (who raised this question)
cc @gigony (who may know the answer)

**Comment by jakirkham** [comment]
Maybe a follow up item here is adding this to the docs

**Comment by gigony** [comment]
@jakirkham @thewtex

The dimension order of the origin is crucial for 3D images, such as CT/MRI images.

There has been some discussion on origin, spacing, and direction at ITK's board: https://discourse.itk.org/t/images-in-physical-space-in-python/2124/15.
I don't recall the conclusion of the discussion, but it appears that establishing a standard for image metadata across Python image libraries is important.

cuCIM operates with a plugin system, where the primary plugin is the cuSlide plugin.
The cuMed plugin is intended to handle CT/MRI volumetric data, but its implementation is not yet planned; currently, we have only a stub plugin implementation for this.

Currently, for Whole Slide Images, the origin metadata is not extensively utilized and is always set to [(0.0, 0.0, 0.0)]
(https://github.com/rapidsai/cucim/blob/39f7c017f9a6eabed39214b68bfbd49b8e5434ad/cpp/plugins/cucim.kit.cuslide/src/cuslide/cuslide.cpp#L216).

**Comment by jakirkham** [comment]
Thanks Gigon! 🙏

When does this `origin` get set in `CuImage` construction? Is this part of the `parse` call here?

https://github.com/rapidsai/cucim/blob/46da10e05be93746910370ea5121447fbb21430e/cpp/src/cuimage.cpp#L186

Or does it happen somewhere else?

**Comment by gigony** [comment]
> Thanks Gigon! 🙏
> 
> When does this `origin` get set in `CuImage` construction? Is this part of the `parse` call here?
> 
> https://github.com/rapidsai/cucim/blob/46da10e05be93746910370ea5121447fbb21430e/cpp/src/cuimage.cpp#L186
> 
> Or does it happen somewhere else?

@jakirkham 
Correct, the method call would call `static bool CUCIM_ABI parser_parse(CuCIMFileHandle_ptr handle_ptr, cucim::io::format::ImageMetadataDesc* out_metadata_desc)` method here ( https://github.com/rapidsai/cucim/blob/39f7c017f9a6eabed39214b68bfbd49b8e5434ad/cpp/plugins/cucim.kit.cuslide/src/cuslide/cuslide.cpp#L216)
